src/core/tests/Test_StatusHandlerTruncation.py

```diff
@@ -89,7 +89,7 @@ def test_only_assessment_patches_over_size_limit_truncated(self):
         truncated status file byte size: 126kb. """
 
         self.__test_scenario = 'assessment_only'
-        self.__patch_count_assessment = 100000
+        self.__patch_count_assessment = 10000
```

Comment:
Why isn't this part of the configuration?

Reply:
because the input patches size varies by unit test by scenarios and that's a class level var will be used in assertion logic